llm-examples/falcon/falcon-7b-instruct-inference.py

```diff
@@ -299,19 +299,13 @@ def predict(self, context, model_input):
 # MAGIC ### Load the model from model registry
 # MAGIC Assume that the below code is run separately or after the memory cache is cleared.
 # MAGIC
-# MAGIC If it's run in a separate notebook, make sure to install the Python libraries before loading from MLflow.
-# MAGIC ```
-# MAGIC %pip install -q -U torch==2.0.1
-# MAGIC %pip install -q einops==0.6.1
-# MAGIC ```
+# MAGIC You can clear the GPU memory through the "Detach & Re-attach" button.
 
 # COMMAND ----------
 
-# Clear GPU memory
-
-from numba import cuda
-device = cuda.get_current_device()
-device.reset()
+# Install the Python packages after the status of the notebook was cleared
+%pip install -q -U torch==2.0.1
+%pip install -q einops==0.6.1
```

Comment:
why you need to reinstall here?

Reply:
Because if it's after detach & re-attaching the notebook, the Python packages would be cleared.